integrations/google-calendar/syncs/events.ts

```
@@ -1,25 +1,34 @@
 import type { NangoSync, GoogleCalendarEvent, ProxyConfiguration } from '../../models';
+import type { GoogleCalendarEventsResponse } from '../types';
+import { toEvent } from '../mappers/to-event.js';
 
 export default async function fetchData(nango: NangoSync): Promise<void> {
-    const oneMonthAgo = new Date();
-    oneMonthAgo.setMonth(oneMonthAgo.getMonth() - 1);
-    oneMonthAgo.setHours(0, 0, 0, 0);
-    const timeMin = oneMonthAgo.toISOString();
-
-    const maxResults = '100';
+    let endpoint = 'calendar/v3/calendars/primary/events';
```

ESLint

[error] 'endpoint' is never reassigned. Use 'const' instead.

(prefer-const)